### PR TITLE
Support to running on the docker into eks

### PR DESCRIPTION
**Pull request by 1dennispark**
For EKS run to docker, we need rocksdb as a self-compiled. It was installed dependencies by [grocksdb install script](https://github.com/linxGnu/grocksdb/blob/master/build.sh). The grocksdb with the dependencies is not working on the AlpineOS.

This PR had solve this problem since EKS run-time by using of self-compiled rocksdb.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,26 +25,24 @@ RUN uname -a
...
-RUN chown -R shapleuser /shapleq
+RUN chown -R shapleuser /shapleq &&\
+    ls -al /shapleq/config/broker
 USER shapleuser:shapleuser
 EXPOSE 1101
-ENV ZK_ADDR=127.0.0.1
-ENV ZK_PORT=2181
-ENTRYPOINT ["bash", "-c", "./start-shapleq.sh $ZK_ADDR $ZK_PORT /shapleq/config/broker/config.yml"]
+ENV ZK_QUORUM="127.0.0.1:2181" HOME=/shapleq
+ENTRYPOINT ["bash", "-c", "./start-shapleq.sh $ZK_QUORUM /shapleq/config/broker/config.yml"]
```
Comment (elon0823) [inline Dockerfile:48]: The ZK_QUORUM should be FQDN of zk service running on EKS. And hostname in broker/config.yml should be FQDN of broker service too.

**Review by elon0823 — CHANGES_REQUESTED** [review]
It seems good to add configuration file for production to set timeout as 0(infite), set hostname, zk_quorum as FQDN of EKS service and so on.

File: Dockerfile (hunk 1 above)
Comment (1dennispark) [reply Dockerfile:47]: No. this Dockerfile is considered that using for general purpose. The ZK_QUORUM config value is changed by EKS setting, like deployment.yaml or statefulset.yaml, on a project-self

Comment (elon0823) [reply Dockerfile:47]: You are right.

File: Makefile [hunk 2]
```diff
@@ -93,14 +93,14 @@ build-rocksdb: $(LIB_ROCKSDB)
 
 $(BROKER_BIN): $(LIB_ROCKSDB)
 ifdef linux-os-host
-	CGO_ENABLED=1 CGO_CFLAGS="-I$(ROCKSDB_INCLUDE_DIR)" \
-	CGO_LDFLAGS="-L$(ROCKSDB_LIB_DIR)" \
+	CGO_ENABLED=1 CGO_CFLAGS="${CGO_CFLAGS}-I$(ROCKSDB_INCLUDE_DIR)" \
```
Comment (elon0823) [inline Makefile:96]: Space required between `${CGO_CFLAGS}` and `-I$(ROCKSDB_INCLUDE_DIR)`

File: Makefile [hunk 3]
```diff
@@ -93,14 +93,14 @@ build-rocksdb: $(LIB_ROCKSDB)
...
+	CGO_LDFLAGS="${CGO_LDFLAGS}-L$(ROCKSDB_LIB_DIR)" \
```
Comment (elon0823) [inline Makefile:97]: Space required between `${CGO_LDFLAGS}` and `-L$(ROCKSDB_LIB_DIR)`

**Review by elon0823 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (1dennispark) [reply Makefile]: Ok. thanks.

File: Makefile (hunk 3 above)
Comment (1dennispark) [reply Makefile]: ditto.

File: start-shapleq.sh [hunk 4]
```diff
@@ -2,17 +2,7 @@
...
-
-
-while [ "$res" != "imok" ]
-do
-  echo "waiting for zookeeper run" 
-  sleep 1
-  res=$(echo ruok | nc $zkhost $zkport)
-done
+zkquorum=$1
+configpath=$2
```
Comment (elon0823) [inline start-shapleq.sh:6]: Is there no need to add hostname as an argument?

Comment (1dennispark) [reply start-shapleq.sh:6]: The script file is existed only for compatibility.

Therefore, We need to change docker process for general purpose. I think that its work is not focused on this PR. To discussion of its work, what is the docker process for general purpose, then the script file is changed.

**Review by elon0823 — APPROVED** [review]

**Pull request merged**